docs/source-pytorch/common/trainer.rst

```diff
@@ -158,7 +169,7 @@ Example::
 
     seed_everything(42, workers=True)
     # sets seeds for numpy, torch and python.random.
-    model = Model()
+    model = LightningTransformer(vocab_size=dataset.vocab_size)
```

Comment:
The main message here is reproducibility, but the details of the model here distract. The previous line was clearer.

Reply:
changed in [05459ca](https://github.com/Lightning-AI/lightning/pull/18452/commits/05459caf327ed834c88f61e1dbefcc4a0a4b240f)